corva/network/api.py

```diff
@@ -54,20 +54,25 @@ def _init_session(api_key: str, app_name: str, max_retries: int, allowed_methods
 
         return session
 
-    def get(self, path: str, **kwargs):
-        return self._request('GET', path, **kwargs)
+    @property
+    def get(self):
+        return self._request('GET')
```

Comment:
why are all methods are properties?

Reply:
They were made properties to get proper parameter information:

Before:
![image](https://user-images.githubusercontent.com/72026031/104896395-b659f700-597f-11eb-83bd-371e16938a2f.png)


After:
![image](https://user-images.githubusercontent.com/72026031/104896289-96c2ce80-597f-11eb-881c-1993d9363834.png)

Reply:
it does not sound like a good idea to use language features to force IDE to behave in the way you want it to.
Since `get` is a method of the `Api` class - it's ok to have it as the first parameter in parameters